modules/test/playwright/tests/export-import-web/import.spec.ts

```diff
@@ -606,3 +618,331 @@ test('can see corresponding elements at site level', async ({
 		exportImportPage.page.getByRole('group', {name: 'Pages'})
 	).toBeVisible();
 });
+
+test(
+	'can import custom object entries with original creator, and creator user does exist in the current environment',
+	{
+		tag: '@LPD-43217',
+	},
+	async ({
+		apiHelpers,
+		applicationsMenuPage,
+		companyExportImportPage,
+		editUserPage,
+		page,
+		usersAndOrganizationsPage,
+		viewObjectDefinitionsPage,
+	}) => {
+		const newObjectDefinition =
+			await apiHelpers.objectAdmin.postRandomObjectDefinition({
+				objectFolderExternalReferenceCode: 'default',
+				status: {code: 0},
+			});
+
+		const user = await apiHelpers.headlessAdminUser.postUserAccount();
+
+		userData[user.alternateName] = {
+			name: user.givenName,
+			password: 'test',
+			surname: user.familyName,
+		};
+
+		await applicationsMenuPage.goToUsersAndOrganizations();
+		await usersAndOrganizationsPage.goToUser(user.name);
+		await editUserPage.rolesLink.click();
+
+		await editUserPage.selectRegularRolesButton.click();
+
+		await editUserPage
+			.selectRegularRolesChooseButton('Administrator')
+			.click();
+
+		await editUserPage.selectRegularRolesButton.click();
+
+		await editUserPage.selectRegularRolesChooseButton('Power User').click();
+
+		await editUserPage.saveButton.click();
```

Comment:
is it possible to use apiHelper to add roles to users? Maybe we could move it to be performed as well in the API layer